tests/integration/prepare/test_ncep_reanalysis_surface_pr_wtr.py

```diff
@@ -0,0 +1,663 @@
+# pylint: disable=E501
+from functools import partial
+from pprint import pformat
+import yaml
+from deepdiff import DeepDiff
+from pathlib import Path
+
+from .common import run_prepare_cli
+from eodatasets.prepare import ncep_reanalysis_surface_pr_wtr
+
+NCEP_PR_WTR_FILE: Path = Path(__file__).parent / 'data' / 'ncep-pr-wtr/pr_wtr.eatm.2018.test.nc'
+
+_diff = partial(DeepDiff, significant_digits=6)
+
+
+def test_prepare_ncep_reanalysis1_pr_wtr(tmpdir):
+    output_path = Path(tmpdir)
+    expected_metadata_path = output_path / 'pr_wtr.eatm.2018.test-metadata.yaml'
+
+    expected_doc = [
+        {
+            "creation_dt": "2019-04-17T01:14:42.954377+00:00",
+            "description": "",
+            "extent": {
+                "center_dt": "2018-01-01T00:00:00+00:00",
+                "coord": {
+                    "ll": {
+                        "lat": -91.25,
+                        "lon": -1.25
+                    },
+                    "lr": {
+                        "lat": -91.25,
+                        "lon": 358.75
+                    },
+                    "ul": {
+                        "lat": 91.25,
+                        "lon": -1.25
+                    },
+                    "ur": {
+                        "lat": 91.25,
+                        "lon": 358.75
+                    }
+                }
+            },
+            "format": {
+                "name": "netCDF"
+            },
+            "grid_spatial": {
+                "projection": {
+                    "geo_ref_points": {
+                        "ll": {
+                            "x": -1.25,
+                            "y": -91.25
+                        },
+                        "lr": {
+                            "x": 358.75,
+                            "y": -91.25
+                        },
+                        "ul": {
+                            "x": -1.25,
+                            "y": 91.25
+                        },
+                        "ur": {
+                            "x": 358.75,
+                            "y": 91.25
+                        }
+                    },
+                    "spatial_reference": (
+                        'GEOGCS["Hu Tzu Shan 1950",DATUM["Hu_Tzu_Shan_1950",SPHEROID'
```

Comment:
I strongly prefer just the epsg "epsg:4236" rather than the full WKT, unless there's a need?